features/fetchlater.yml

```
@@ -0,0 +1,7 @@
+name: fetchLater
+description: The `fetchLater()` method requests a deferred fetch sent at an unknown time. The browser chooses a reliable time to send the request, ideally when the document is unloaded, and ignores the response. This API is useful for sending beacons, that is sending data to a server, without expecting a particular response.
```

To work around the char limit, maybe this?

```suggestion
description: The `fetchLater()` method requests a deferred fetch sent at an unknown time. The browser chooses a reliable time to send the request, ideally when the document is unloaded, and ignores the response. This API is useful for sending beacons without expecting a particular response.
```